scalableminds/webknossos
# Add brush size settings and shortcuts

This PR adds a second keyboard shortcut to volume tracing to increase/decrease the brush size. It also added it to the settings in the settings tab on the left side.

### URL of deployed dev instance (used for testing):
- [https://addbrushsizesettingsandshortcuts.webknossos.xyz](https://addbrushsizesettingsandshortcuts.webknossos.xyz)

### Steps to test:
- open volume tracing and select the brush mode
- the brush should resize according to the settings set on the left tab, and according to the shortcuts (shift + mouse wheel/ shift + i,o)
- the settings option for the brush size should only appear in volume tracing

### Issues:
- fixes #2328

------
- ~~~[ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~~
- ~~~[ ] Needs datastore update after deployment~~~
